projects/aca-shared/src/lib/components/info-drawer/info-drawer.component.html

```diff
@@ -2,11 +2,11 @@
   <mat-progress-bar mode="indeterminate" [attr.aria-label]="'APP.INFO_DRAWER.DATA_LOADING' | translate"></mat-progress-bar>
 </div>
 <ng-container *ngIf="!isLoading && !!displayNode">
-  <adf-info-drawer [title]="'APP.INFO_DRAWER.TITLE'" cdkTrapFocus cdkTrapFocusAutoCapture>
+  <adf-info-drawer [drawerIcon]="node.entry"  [title]="node?.entry ? node.entry.name : 'APP.INFO_DRAWER.TITLE'" cdkTrapFocus cdkTrapFocusAutoCapture>
```

Comment:
what is `drawerIcon` and why it requires `node.entry`?

Reply:
We need to display the node icon alongside the name, so I've incorporated that into the design.
Additionally, I've now updated the name to "nodeIcon."